smartmodule/examples/jolt/README.md

```diff
@@ -0,0 +1,82 @@
+# JSON to JSON transformation SmartModule
+
+This is a `map` type SmartModule that transforms JSON records using [Fluvio Jolt][1] library.
```

Comment:
Jolt should move to jolt repo. This work is already part of @ozgrakkurt

Reply:
This is a combination of `jolt` and `jolt-sql`, so it's an implementation of Jolt in Fluvio, hence an example. I still think it belongs here.

Reply:
Jolt related code should go into jolt repo otherwise it will be hard to find related items